packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/ManageSubscription/stripe.tsx

```diff
@@ -30,20 +24,21 @@ export const Stripe: React.FC<{ hasActiveTrial: boolean }> = ({
               event_source: 'UI',
             });
           }
-          createCustomerPortal();
+
+          openCancelSubscriptionModal();
         }}
         size={3}
         variant="active"
       >
-        {loading ? 'Loading...' : ctaText}
+        {hasActiveTeamTrial ? 'Cancel trial' : 'Manage subscription'}
```

Comment:
We should change the CTA from `Manage subscription` to `Cancel subscription`. Otherwise it feels weird to press `Manage subscription` and to be presented with a modal telling you about cancellation when maybe you want to increase the number of seats or upgrade to a different tier (in the future). Or maybe we can show two links that both go to the same portal for now, but make one explicit about cancelling the subscription and one about managing?

Reply:
Addressed it here: https://github.com/codesandbox/codesandbox-client/pull/7195/commits/de6c7d4b304ee205ba0edfa1ef7ce3339da28d1a.